src/Adapter/Redis/Tests/CreateArrayPoolTrait.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+/*
+ * This file is part of php-cache organization.
+ *
+ * (c) 2015 Aaron Scherer <aequasi@gmail.com>, Tobias Nyholm <tobias.nyholm@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Cache\Adapter\Redis\Tests;
+
+use Cache\Adapter\Redis\RedisCachePool;
+
+trait CreateArrayPoolTrait
```

Comment:
Can you change the traits names to something like this CreateRedisArrayCachePoolTrait